tests/phpunit/tests/image/functions.php

```diff
@@ -726,6 +726,134 @@ public function test_wp_generate_attachment_metadata_pdf() {
 		remove_filter( 'wp_upload_image_mime_transforms', '__return_empty_array' );
 	}
 
+	/**
+	 * @ticket 55443
+	 */
+	public function test_wp_generate_attachment_metadata_pdf_webp_mime() {
+		if ( ! wp_image_editor_supports( array( 'mime_type' => 'application/pdf' ) ) ) {
+			$this->markTestSkipped( 'Rendering PDFs is not supported on this system.' );
+		}
```

Comment:
Along with this, we should first check if the editor supports WebP or not because we added this test for WebP.

```
if ( ! wp_image_editor_supports( array( 'mime_type' => 'image/webp' ) ) ) {
	$this->markTestSkipped( 'This test requires WebP support.' );
}
```